### ISSUE TITLE
sqlint disabled. 'cls.defaults' is mandatory and MUST be a dict.

### ISSUE THREAD

**Issue by tpilewicz** [trezona-lecomte/SublimeLinter-contrib-sqlint#4]
Hey, don't know if you're still maintaining this, but SublimeLinter doesn't validate the Sqlint class anymore. Two error messages are raised:
```
SublimeLinter: ERROR:
=====================
sqlint: Defining 'cls.syntax' has no effect anymore. Use http://www.sublimelinter.com/en/stable/linter_settings.html#selector instead.


SublimeLinter: ERROR:
=====================
sqlint disabled. 'cls.defaults' is mandatory and MUST be a dict.
```

I think replacing `syntax = 'sql'` with `defaults = {'selector': 'source.sql'}` would fix it?

**Comment by tpilewicz** [comment]
#5 to fix this, tested locally

**Comment by trezona-lecomte** [comment]
Fixed by @tpilewicz in #5, tested locally.
